docs/how-to/create-a-router/configure-ensembler.md

```diff
@@ -13,6 +13,14 @@ The router will return a response from the route configured to act as the final
 
 ![](../../.gitbook/assets/nop_ensembler_config.png)
 
+## Standard Ensembler
+
+For experiment engines configured to work with standard ensemblers (i.e., Standard Experiment Engines with experiment selection enabled), the router will return a response from from one of the routes based on the configured mapping between routes and experiment treatments. At run time, the treatment returned by the engine will be used to select the corresponding route’s response.
+
+In addition, a fallback route may be configured whose results will be used at runtime when the call to the experiment engine fails or if there does not exist a route mapping for the treatment generated by the experiment engine.
```

Comment:
nit: `or if a route mapping for the treatment generated by the experiment engine does not exist`

Reply:
My way sounds fancier. :P But I'll take your suggestion.